Sequence.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class Sequence {
+  public static void main(String[] args) {
+    boolean test = true; 
+    Scanner input = new Scanner(System.in);
+    System.out.println("Enter numerical sequence /please use space while entering/:");
+    String gaps[] = s.split(" "); 	
+    int seq[] = new int[gaps.length];  
+    try {
+      for (int i = 0; i < gaps.length; i++) {
+        seq[i] = Integer.parseInt(gaps[i]);

```

Comment:
you don't actually need this temporary array - try to find solution without tmp int array